docs/guides/distribution.md

```diff
@@ -3,37 +3,37 @@
 Distributing your games
 ===================
 
-DOME is designed to be cross-platform, and so the same Wren game files and assets should work across Windows, Mac and Linux. On Mac and Linux, the SDL2 shared library will need to be provided.
+DOME is designed to be cross-platform, and so the same Wren code and assets should work across Windows, Mac and Linux.
 
 ## Basic Packaging
 The easiest way to share a game you've made is to place a DOME binary, your source code and game assets into a single zip file. Do this once for each platform you wish to support, and share those zip files with your users.
+If you are providing your own DOME binary, rather than a pre-compiled one, you will need to have access to the shared SDL2 library.
 
-## NEST - Easy bundling
+## nest - Easy bundling
 
-_(Please note: NEST is still in development, and has to be built from source)_
+For easy distribution, you can package all of your application code and resources into a single `.egg` file using the `nest` tool built into DOME. DOME automatically plays any file named `game.egg` in the current working directory.
 
-For easy distribution, you can package all of your games resources into a single `.egg` file using a tool called [NEST](https://github.com/domeengine/nest). DOME automatically plays any file named `game.egg` in the current working directory.
+If you use a `.egg` file, DOME expects your game to start from a `main.wren` in the base of the bundle, as its entry point.
```

Comment:
When running a ".egg" file, DOME looks for a "main.wren" file in the base directory of the bundle, as the entry point for execution.